README.md

```diff
@@ -410,7 +410,19 @@ mutation($id: ID!) {
   { id: 5 }
 );
 ```
+### use urql without using components
 
+While the goal of this project is to work nicely with react, sometimes you just need to do a mutation or a query without using any components. In this case you can call [client.executeQuery](src/components/client.tsx#L188) directly.
+
+Example:
+
+```javascript
+let variables = {"q": "Did I do that?"}
+let skipCache = true //set this to true if you don't need cache
+client.executeQuery(query(myQuery, variables), skipCache).then(function(data) {
+  console.log(data);
+});
+```
```

Comment:
This snippet needs some refactoring as well, e.g. change `let` to `const`. Also, we don't need to have a dummy variables object here, we have already documented `query()`. 

Although I don't know if the core team members goal is to document this, as it can be used as a bad practice, most of the times.